mathesar_ui/src/component-library/confirmation/Confirmation.svelte

```diff
@@ -34,6 +34,7 @@
     } catch (error) {
       // @ts-ignore: https://github.com/centerofci/mathesar/issues/1055
       onError(error);
+      modal.close();
```

Comment:
The modal seems to close on an error which may not want in several circumstances.

This comes from a community PR: https://github.com/centerofci/mathesar/pull/2632

We need to look at this change.

Reply:
Thanks for noticing this, @pavish. I've opened #2729 to fix this.